.config/vscode-recommended/launch.json

```diff
@@ -0,0 +1,21 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "type": "lldb",
+      "request": "launch",
+      "name": "prqlc _a.prql",
```

Comment:
What is this?

Reply:
But what's `_a.prql`?

Reply:
A local file containing the query you are currently working on.

Reply:
I see, OK

Reply:
Mine usually has a bunch of different cases commented out, so I can enable them quickly.